ui/tests/integration/components/info-table-row-test.js

```diff
@@ -70,4 +75,27 @@ module('Integration | Component | InfoTableItem', function(hooks) {
 
     assert.dom('[data-test-item="array"]').hasText('valueArray', 'Confirm link with item value exist');
   });
+
+  test('it renders a dash (-) if a label and/or value do not exist', async function(assert) {
```

Comment:
We should maybe have one more tests that confirms that any block content overrides any value content